### PR TITLE
Introducing utility function libs

### PR DESCRIPTION
**Pull request by xandrd**
Addressing Issue #629

### CODE REVIEW

File: pyspedas/utilities/libs.py
Comment (jameswilburlewis) [inline pyspedas/utilities/libs.py:1]: Could you have it search both pyspedas and pytplot by default?

Comment (xandrd) [reply pyspedas/utilities/libs.py:1]: Search by pytplot is added. I also added corresponding tests.

Comment (xandrd) [reply pyspedas/utilities/libs.py:1]: The branch is ready to be merged into `master`

**Comment by jameswilburlewis** [comment]
Looks good, thanks!

**Pull request merged**